src/Core/Validation/FieldValidation/FieldValidatorsTrait.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace SilverStripe\Core\Validation\FieldValidation;
+
+use RuntimeException;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Core\Config\Configurable;
+use SilverStripe\Core\Validation\FieldValidation\FieldValidationInterface;
+use SilverStripe\Core\Validation\ValidationResult;
+use SilverStripe\Forms\FormField;
+
+trait FieldValidatorsTrait
```

Comment:
There's not really any consistency for naming traits in Silverstripe, over 25% of them are suffixed with Trait. Some of them are suffixed with 'ible/able', some of them are suffixed with 'Aware', one of them is called 'BlogObject' :smile: 

My brain finds it far easier to simply put the 'Trait' suffix on traits, so will leave as is

HasFieldValidator sounds wrong, the 'Has' prefix makes it sound like it's a method that returns a boolean

I've renamed it simply to `FieldValidationTrait` so that it matches `FieldValidationInterface` since the two need to be used together

Reply:
I don't agree about the name but at this point I just want to merge it so we'll leave as is.